docs/deprecations.md

```diff
@@ -10,6 +10,14 @@ layout: Doc
 
 To disable specific deprecations set `SLS_DEPRECATION_DISABLE` environment variable. Setting `SLS_DEPRECATION_DISABLE=*` will disable all deprecations. If you want to disable specific deprecations set `SLS_DEPRECATION_DISABLE=CODE1,CODE2`.
 
+<a name="VARIABLES_ERROR_ON_UNRESOLVED"><div>&nbsp;</div></a>
```

Comment:
Recently, there was a slight change to how we should add deprecations - as you can see on examples below yours, there's an additional line with `Deprecation code` - could you add that one as well?

Reply:
Done